src/map/skill.c

```diff
@@ -8619,8 +8619,9 @@ static int skill_castend_nodamage_id(struct block_list *src, struct block_list *
 			int splash;
 			if (flag&1 || (splash = skill->get_splash(skill_id, skill_lv)) < 1) {
 				int i;
-				if( sd && dstsd && !map_flag_vs(sd->bl.m)
-					&& (sd->status.party_id == 0 || sd->status.party_id != dstsd->status.party_id) ) {
+				if (sd && dstsd && !map_flag_vs(sd->bl.m)
```

Comment:
since this was modified, go extra mile and expand these first two conditions.

Reply:
True. Stuff like `sd` should be `sd != NULL`, same for dstsd.

Reply:
These checks are actually not needed as all of them are previously checked already. The only detail left is guild mates.
Currently, the flag GuildOnly in skill_db allows casting it on guild mates in and outside of gvg, I'm thinking on adding a new flag `GuildPvpOnly` it applies to guild mates only in pvp scenarios.

Reply:
I've finally discarded the idea of adding yet another flag for 1 or 2 skills that will make use of it.
Fixes related to these comments, done.